boxes.yaml.example

```diff
@@ -44,6 +44,7 @@ centos7-bats-ci:
       bats_tests:
         - "swapfile.bats"
         - "fb-install-katello.bats"
```

Comment:
For now I'd suggest a haveged role rather than a more global setup role which may get parameters. Start simple.